src/mgr/MgrStandby.cc

```diff
@@ -157,7 +157,7 @@ void MgrStandby::send_beacon()
   // as available in the map)
   bool available = active_mgr != nullptr && active_mgr->is_initialized();
 
-  auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
+  auto addr = available ? active_mgr->get_server_addr() : monc.get_my_addr();
```

Comment:
The risk here seems to be that someone might assume the standby addr will become the active addr when that's not actually the case.  That aside I do'nt have a philosophical problem with this